bundle/Resources/config/menu_plugins.yml

```diff
@@ -28,3 +28,11 @@ services:
         public: false
         tags:
             - { name: "netgen_admin_ui.menu_plugin", priority: -20 }
+
+    netgen_admin_ui.menu_plugin.information_collection:
+        class: Netgen\Bundle\AdminUIBundle\MenuPlugin\InformationCollectionMenuPlugin
+        public: false
+        arguments:
+            - "%kernel.bundles%"
+        tags:
+            - { name: "netgen_admin_ui.menu_plugin", priority: -1 }
```

Comment:
Please use priorities different from other plugins, and add at least 10 between them.